packages/amazonq/test/e2e/amazonq/review.test.ts

```diff
@@ -0,0 +1,307 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Zuo
+ */
+
+import assert from 'assert'
+import vscode from 'vscode'
+import { qTestingFramework } from './framework/framework'
+import sinon from 'sinon'
+import { Messenger } from './framework/messenger'
+import { registerAuthHook, using, closeAllEditors } from 'aws-core-vscode/test'
+import { loginToIdC } from './utils/setup'
+import { codewhispererDiagnosticSourceLabel } from 'aws-core-vscode/codewhisperer'
+import path from 'path'
+
+function getWorkspaceFolder(): string {
+    return (
```

Comment:
I've seen a couple teams do the same thing, do we not have a util? Also, we can expect the first workspace folder to load, otherwise the test setup didn't complete successfully